site/docs/3_manage_nifi/1_manage_clusters/1_deploy_cluster/2_nodes_configuration.md

```diff
@@ -289,10 +289,10 @@ In this example if we have the `nifi.sensitive.props.key` key defined in the sec
 
 ### Overridable configurations
 
-Here is the list of configuration that you can override for NiFi :
+Here is the list of configuration that you can override for NiFi:
 - [nifi.properties](https://github.com/konpyutaika/nifikop/blob/master/pkg/resources/templates/config/nifi_properties.go)
 - [zookeeper.properties](https://github.com/konpyutaika/nifikop/blob/master/pkg/resources/templates/config/zookeeper_properties.go)
-- [bootstrap.properties](https://github.com/konpyutaika/nifikop/blob/master/pkg/resources/templates/config/bootstrap_properties.go)
+- [bootstrap.conf](https://github.com/konpyutaika/nifikop/blob/master/pkg/resources/templates/config/bootstrap_properties.go)
```

Comment:
should this URL change to bootstrap.conf as well?

Reply:
I had to rename a file, so I renamed some functions too.